src-js/site/main.js

```diff
@@ -316,6 +317,7 @@ $(document).ready(function () {
             window.AnalyticsClient.trackAction('docs-share-button-click', {
               page: location.pathname,
               success: true,
+              section: url.substring(url.indexOf('#')),
```

Comment:
Looks good but I don't think that will work when clicking on the main document title since there is no section? For example in https://circleci.com/docs/2.0/migrating-from-buildkite/ :

![Screen Shot 2021-11-01 at 4 32 16 PM](https://user-images.githubusercontent.com/1449325/139755634-bf4d328e-6aa4-4e59-8cb0-471e34ffb3cd.png)

Reply:
Thats a great point. I can try to find a way to account for that (or use the full url path) but as we also know the `page`, do you think this could work since we know `#` will denote the title link for the page that we are on when we review the data?